style/pi-hole.css

```diff
@@ -1471,3 +1471,10 @@ table.dataTable tbody > tr > .selected {
     text-align: right;
   }
 }
+
+textarea.field-sizing-content {
+  min-block-size: 3.5rlh;
+  max-block-size: 20rlh;
```

Comment:
```suggestion
```

Removing `max-block-size` will allow the textarea to grow just like the other input fields (form elements are using `width: 100%`).